configs/projects/_shared-agent-settings.rb

```diff
@@ -99,7 +99,7 @@
 platform_triple = "ppc64le-redhat-linux" if platform.architecture == "ppc64le"
 platform_triple = "powerpc64le-suse-linux" if platform.architecture == "ppc64le" && platform.name =~ /^sles-/
 platform_triple = "powerpc64le-linux-gnu" if platform.architecture == "ppc64el"
-platform_triple = "arm-linux-gnueabihf" if platform.name == 'debian-8-armhf'
+platform_triple = "arm-linux-gnueabihf" if platform.name =~ /debian-[\d?\d]-armhf/
```

Comment:
Same as my other comment, we can use `platform.architecture =~ "armhf"` as a condition.